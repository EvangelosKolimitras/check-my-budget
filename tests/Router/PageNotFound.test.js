import React from "react"
import toJson from "enzyme-to-json"
import PageNotFound from "../../src/scripts/js/Router/PageNotFound"
import { shallow, configure } from "enzyme"
import Adapter from "enzyme-adapter-react-16"
configure({ adapter: new Adapter() })

describe("Routers", () => {
    describe("the PageNotFound router", () => {
        it("should render PageNotFound correctly", () => {
            const wrapper = shallow(<PageNotFound />)
            expect(toJson(wrapper)).toMatchSnapshot()
        })
    })
})
