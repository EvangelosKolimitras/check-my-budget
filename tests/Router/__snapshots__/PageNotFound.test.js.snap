// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routers the PageNotFound router should render PageNotFound correctly 1`] = `
<div>
  <h1>
    Page Not Found - 404
  </h1>
</div>
`;
